service/network/responses.go

```diff
@@ -0,0 +1,46 @@
+package network
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"bootcamp/domain/model"
+)
+
+func setHeaders(w http.ResponseWriter) http.ResponseWriter {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	return w
+}
+
+func validateError(w http.ResponseWriter, err error) {
+	if err != nil {
```

Comment:
This check is a bit redundant, right? Maybe we could get rid of this `if`

Reply:
*